core/src/com/group66/game/screens/SettingsScreen.java

```diff
@@ -0,0 +1,214 @@
+
+        skin = new Skin();
+
+        // Store the default libgdx font under the name "default".
+        BitmapFont bfont = new BitmapFont();
+        skin.add("default", bfont);
+
+        // Generate a 1x1 white texture and store it in the skin named "white".
+        Pixmap pixmap = new Pixmap(BUTTON_WIDTH, BUTTON_HEIGHT, Format.RGBA8888);
+        pixmap.setColor(Color.WHITE);
+        pixmap.fill();
+        skin.add("white", new Texture(pixmap));
+
+        // Configure a TextButtonStyle and name it "default". Skin resources are
+        // stored by type, so this doesn't overwrite the font.
+        TextButtonStyle textButtonStyle = new TextButtonStyle();
+        textButtonStyle.up = skin.newDrawable("white", Color.GRAY);
+        textButtonStyle.down = skin.newDrawable("white", Color.DARK_GRAY);
+        textButtonStyle.checked = skin.newDrawable("white", Color.BLUE);
+        textButtonStyle.over = skin.newDrawable("white", Color.LIGHT_GRAY);
+        textButtonStyle.font = skin.getFont("default");
+        skin.add("default", textButtonStyle);
+
+        //all magic numbers in this section are offsets values adjusted to get better looks
+        int yoffset = Gdx.graphics.getHeight() / 2 + 2 * (BUTTON_HEIGHT + BUTTON_SPACING) - 75;
+        
+        TextButton easyButton = new TextButton("Easy", textButtonStyle);
+        easyButton.setPosition((Gdx.graphics.getWidth() - BUTTON_WIDTH - 300) / 2, 
+                yoffset - BUTTON_HEIGHT - BUTTON_SPACING);
+        
+        TextButton mediumButton = new TextButton("Medium", textButtonStyle);
+        mediumButton.setPosition((Gdx.graphics.getWidth() - BUTTON_WIDTH) / 2,
+                yoffset - (BUTTON_HEIGHT + BUTTON_SPACING));
+        
+        TextButton hardButton = new TextButton("Hard!", textButtonStyle);
+        hardButton.setPosition((Gdx.graphics.getWidth() - BUTTON_WIDTH + 300) / 2,
+                yoffset - (BUTTON_HEIGHT + BUTTON_SPACING));
+
+        TextButton menuButton = new TextButton("Menu", textButtonStyle);
+        menuButton.setPosition((Gdx.graphics.getWidth() - BUTTON_WIDTH) / 2,
+                yoffset - 2 * (BUTTON_HEIGHT + BUTTON_SPACING));
+        
+        stage.addActor(easyButton);
+        stage.addActor(mediumButton);
+        stage.addActor(hardButton);
+        stage.addActor(menuButton);
+
+        // Add a listener to the button. ChangeListener is fired when the
+        // button's checked state changes, eg when clicked,
+        // Button#setChecked() is called, via a key press, etc. If the
+        // event.cancel() is called, the checked state will be reverted.
+        // ClickListener could have been used, but would only fire when clicked.
+        // Also, canceling a ClickListener event won't
+        // revert the checked state.
+        easyButton.addListener(new ChangeListener() {
+            public void changed(ChangeEvent event, Actor actor) {
+                DifficultyManager.setDifficulty("easy");
```

Comment:
Yah it's generally better for user experience.
After merging your code, I think I'll keep it as it is because he might want to set the difficulty and then choose split screen or multiplayer, for example.